source/localizable/analytics/libraries.html.md.erb

```diff
@@ -0,0 +1,48 @@
+---
+parent: analytics
+title: libraries
+---
+
+# Libraries <%= edit_link %>
+
+Interested in integrating your eshop with <%= flavor.capitalize %> Analytics?
+
+You may want to use one of our existing libraries, that are developed and supported by us. No need to write any code!
+
+<%= partial 'partials/toc' %>
+
+## Wordpress / WooCommerce
+
+You can find the plugin <%= link_to 'here', 'https://wordpress.org/plugins/skroutz-analytics-woocommerce/', target: '_blank' %>.
+
+Requires to have Wordpress 4.0 or higher and the <%= link_to 'WooCommerce plugin', 'https://wordpress.org/plugins/woocommerce/', target: '_blank' %> installed.
+
+### Automatic installation (recommended)
+
+1. Navigate to your wordpress: `Admin panel > Plugins > Add New`
+2. Search this plugin by its name: *Skroutz Analytics WooCommerce*
+3. Install the plugin.
+4. Activate the plugin.
+
+### Manual installation
+
+1. Download the <%= link_to 'plugin file', 'https://wordpress.org/plugins/skroutz-analytics-woocommerce/', target: '_blank' %> to your computer and unzip it.
+2. Using an FTP client, or your hosting control panel, upload the unzipped plugin folder to your WordPress installation’s directory: `wp-content/plugins/`
+3. Activate the plugin through the Plugins menu in WordPress admin.
+
+### Configuration
```

Comment:
Maybe we can avoid having duplicate documentation about config options and point them to the wordpress plugins page doc.